scripts/build_rootfs.sh

```diff
@@ -143,6 +146,14 @@ install_packages() {
 	done
 }
 
+disable_services() {
+    local services="$1"
+    for service in $services; do
+        echo Disabling $service
+        env IPKG_INSTROOT=$instroot sh $instroot/etc/rc.common $instroot/etc/init.d/$service disable
```

Comment:
```suggestion
        test -d $instroot/var/lock || mkdir $instroot/var/lock
        env IPKG_INSTROOT=$instroot bash $instroot/etc/rc.common $instroot/etc/init.d/$service disable
```

The lock directory is missing at this point, and dash (the default shell in Ubuntu among other) doesn't handle the flock command well as seen in the error below. Using bash instead of sh seems to solve it.

```
Disabling sysntpd
flock: 1000: Bad file descriptor
```